src/components/OrderDetailWidget/subcomponents/OrderStatusInfo/OrderStatusInfo.tsx

```diff
@@ -3,23 +3,39 @@ import { useTranslation } from "react-i18next";
 
 import { OrderStatus } from "../../../../types/orderStatus";
 import ExpiryIndicator from "../../../ExpiryIndicator/ExpiryIndicator";
-import { Button, InfoWrapper, StyledIcon } from "./OrderStatusInfo.styles";
+import LoadingSpinner from "../../../LoadingSpinner/LoadingSpinner";
```

Comment:
There's a linting error `'LoadingSpinner' is defined but never used.`

Reply:
Whoops, fixed!